Stripe/PublicHeaders/STPPaymentCardTextField.h

```diff
@@ -109,6 +64,11 @@ IB_DESIGNABLE
 @property(nonatomic, copy, nullable) IBInspectable NSString *cvcPlaceholder;
 
 /**
+ *  The placeholder for the postal code field. Defaults to @"ZIP/Postal".
```

Comment:
Note to self: Change this to make clear that you get either "ZIP" or "Postal" and not "ZIP/Postal"